### DISCUSSION TITLE
[Feature Request] Streamline Azure Log configuration in 8.x

### DISCUSSION THREAD

**Discussion by ehuggz** [grafana/grafana#37352] [Data Sources]
Along with Application Insights and LogAnalytics move to a centralized "Log" option in 8.x came the introduction of selecting resources first by resource group and then by resource.

1. Resource Groups are now listed alphabetically, but the resources are not.
2. Resources include all resources in the Resource Group not just logs.

Requests:
1. List Resources inside ResourceGroups alphabetically
2. Provide some easier way to locate Log Sources such as:
    a. Sorting Application Insights and LogAnalytics resources to top (alphabetically)
    b. Provide filter mechanism by Resource Type, with AppInsights and LogAnalytics at first options
3. Provide search by name for both ResourceGroup and Resources

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
